charts/fybrik/templates/cluster-metadata-config.yaml

```diff
@@ -3,6 +3,10 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: cluster-metadata
+  {{- if eq .Values.global.multiclusetrOrchestrator "razee" }}
```

Comment:
What's wrong with just adding the label if isRazeeEnabled? Why the new value

Reply:
I thought it might be misleading as it is true only when deploying the coordinator cluster.

Reply:
I'm not a fan of adding another global attribute. Consider if just always adding the label is harmless, or otherwise if such flag is more appropriate in the `worker` section or even as `enabled` flag in `razee`.

Reply:
OK I think that adding the label is harmless.